developer/src/kmcmplib/src/CompilerInterfaces.cpp

```diff
@@ -162,6 +129,7 @@ EMSCRIPTEN_BINDINGS(compiler_interface) {
     ;
 
   emscripten::function("kmcmp_compile", &kmcmp_compile);
```

Comment:
Is this line equivalent to the removed `compileKeyboardFile` function from `compiler.ts`?  Otherwise, it looks like we're dropping that function without an equivalent stand-in after these changes.

Reply:
Yes, `compileKeyboardFile`, which was a messy function with legacy and a bunch of bool parameters, is replaced with `kmcmp_compile`, but see #8870 for the end result.